test/end-to-end.js

```diff
@@ -970,6 +976,44 @@ describe('end-to-end', function() {
       });
     });
 
+    it('shows error behavior responses in development mode', function(done) {
```

Comment:
The test name should describe what is the expected behaviour/outcome. E.g. "it includes sensitive error details in development mode".